packages/manager/modules/vps/src/backup-storage/order/translations/Messages_fr_FR.json

```diff
@@ -2,8 +2,8 @@
   "vps_configuration_activate_ftpbackup_title": "Activer l'option Backup storage",
   "vps_configuration_activate_ftpbackup_back_btn_title": "Retour à la liste des Backups storage",
   "vps_configuration_activate_ftpbackup_step1_question": "Commande de l'option Backup storage sur le VPS {{ serviceName }} ?",
-  "vps_configuration_activate_ftpbackup_step1_alert": "Cette option sera facturée au tarif mensuel de <span class=\"bold\">{{ price }} HT</span>",
-  "vps_configuration_activate_ftpbackup_step1_next": "Ensuite, le service sera facturé mensuellement au prix de <strong>{{ price }} HT</strong>.",
+  "vps_configuration_activate_ftpbackup_step1_alert": "Cette option sera facturée au tarif de <span class=\"bold\">{{ price }} HT</span>",
```

Comment:
**thought:** Don't know if it can be applied but could be turned into:
```suggestion
  "vps_configuration_activate_ftpbackup_step1_alert": "Cette option sera facturée au tarif de <strong>{{ price }} HT</strong>",
```